.travis.yml

```diff
@@ -176,3 +177,17 @@ jobs:
       env:
         - MPLBACKEND=ps
         - PYTHON_VERSION=3.7.2
+
+    # Randomized testing and benchmarks
+    - stage: randomized
+      name: Randomized tests
+      <<: *stage_linux
+      script:
+      - python setup.py build_ext --inplace
+      - pip install "qiskit-aer"
+      - make test_randomized
```

Comment:
```suggestion
      - travis_wait make test_randomized
```

Reply:
:+1: Added, I tried bumping the hypothesis verbosity level but it's output isn't all that meaningful for stateful testing.